package.json

```
@@ -64,6 +64,7 @@
     "prettier": "prettier --write 'packages/**/*.ts'",
     "lint": "eslint . --ext .ts",
     "release": "yarn && lerna publish --no-private --force-publish && yarn retype:updateversion",
+    "release:canary": "yarn && lerna publish --canary --no-private --force-publish && yarn retype:updateversion",
```

```suggestion
    "release:canary": "yarn && lerna publish --canary --no-private --force-publish",
```
I'm guessing we don't need retype:updateversion?